src/tests/test_raft_repl_dev.cpp

```diff
@@ -127,16 +131,23 @@ class TestReplicatedDB : public homestore::ReplDevListener {
         LOGINFO("[Replica={}] Received rollback on lsn={}", g_helper->replica_num(), lsn);
     }
 
+    void on_error(ReplServiceError error, const sisl::blob& header, const sisl::blob& key,
+                  cintrusive< repl_req_ctx >& ctx) override {
```

Comment:
assert() as we are not expecting any error.

Reply:
Well, thought about that, but I wanted to extend this for negative check, where propose on leader and check error etc.. So didn't put it here. Later we will add counter and validate that.